Necrosis/Initialize.lua

```diff
@@ -99,7 +99,6 @@ function Necrosis:Initialize(Config)
 		self.Spell[index].ID = nil
 	end
 	self:SpellLocalize()
-	self:CreateSpellList()
```

Comment:
Where is the replacement for this? I see yuou removed the object creation below, what is populating `self.Spell`?

Reply:
Nevermind, I found it's creation in `function Necrosis:SpellSetup()`

Nice